src/dataCollection.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+"""
+@file dataCollection.py .
+
+@author Ayush Saini
+@email ayush@robrosystems.com
+@brief script for Data Collection using ROS Topic
+@version 0.1.0
+@date 2021-08-28
+@copyright Copyright (c) 2021
+"""
+import os
+
+import cv2
+
+from cv_bridge import CvBridge, CvBridgeError
+
+from google.cloud import storage
+
+from robro_cloud.robroBucketImageUploader import RobroBucketImageUploader
+
+import rospy
+
+from sensor_msgs.msg import Image
+
+from std_msgs.msg import String
+
+rospy.init_node('dataCollection', anonymous=True)
+
+proj_name = rospy.get_param('~proj_name')
+ros_topic_name = rospy.get_param('~ros_topic_name')
+image_type = rospy.get_param('~image_type')
+camera_name = rospy.get_param('~camera_name')
+cred_path = rospy.get_param('~cred_path')
+category = 'DEFAULT'
+try:
+    category_param_valid = True
+    category = rospy.get_param('~category')
+except:
+    category_param_valid = False
+
+try:
+    category_topic_valid = True
+    category_topic_name = rospy.get_param('~category_topic_name')
+except:
+    category_topic_valid = False
+
+try:
+    resize_q = True
+    image_size = rospy.get_param('~image_size')
+except:
+    resize_q = False
+
+try:
+    crop_q = True
+    crop_size = rospy.get_param('~crop_size')
+except:
+    crop_q = False
+
+grayscale = rospy.get_param('~grayscale')
+
```

Comment:
put all this variables at one place

Reply:
I did it in order according to the launch file, but okay. Done